features/api-layer/pom.xml

```diff
@@ -131,6 +131,11 @@
             <artifactId>mockito-all</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Nit: This is "grouped" with the test dependencies at the bottom of the pom, could be above that section.